packages/reactivity/src/ref.ts

```diff
@@ -3,11 +3,11 @@ import { OperationTypes } from './operations'
 import { isObject } from '@vue/shared'
 import { reactive } from './reactive'
 
-export const refSymbol = Symbol()
+export const refSymbol = Symbol(__DEV__ ? 'refSymbol' : undefined)
```

Comment:
```suggestion
export const refSymbol = __DEV__ ? Symbol('refSymbol') : Symbol()
```

With your change, we don't really need the `RefSymbol` type.

Reply:
this change will break the tests :/